packages/block-editor/src/hooks/anchor.js

```diff
@@ -71,8 +71,7 @@ function BlockEditAnchorControlPure( { anchor, setAttributes } ) {
 					<>
 						{ __(
 							'Enter a word or two — without spaces — to make a unique web address just for this block, called an “anchor.” Then, you’ll be able to link directly to this section of your page.'
```

Comment:
```suggestion
							'Enter a word or two — without spaces — to make a unique web address just for this block, called an “anchor”. Then, you’ll be able to link directly to this section of your page.'
```

Although it is not directly related to this PR, how about adjusting the position of the period?